### PR TITLE
Make startapp log file location configurable by an env var

### PR DESCRIPTION
**Pull request by brokeh**
So that it is not included in the pushed backups

### CODE REVIEW

**Comment by brokeh** [comment]
No worries. If there's utility in that behaviour then I can change it to be configurable via an env var then.
Does that sound OK?

**Comment by traktuner** [comment]
> No worries. If there's utility in that behaviour then I can change it to be configurable via an env var then. Does that sound OK?

We ask people to provide this logfile to see why there may be an issue with the app not starting/installing, etc and therefore we need a defined path to the logfile and that path should not change.
I think it's best to add the file to the exception list in Backblaze so that it's not backed up.
I put it in the root of C: intentionally that it's not dependent on any wine installation or initialization.

File: README.md [hunk 1]
```diff
@@ -118,6 +118,7 @@ Environment variables can be set by adding one or more arguments `-e "<VAR>=<VAL
 |`VNC_PASSWORD`| Password needed to connect to the application's GUI.  See the [VNC Password](#vnc-password) section for more details. | (unset) |
 |`X11VNC_EXTRA_OPTS`| Extra options to pass to the x11vnc server running in the Docker container.  **WARNING**: For advanced users. Do not use unless you know what you are doing. | (unset) |
 |`ENABLE_CJK_FONT`| When set to `1`, open-source computer font `WenQuanYi Zen Hei` is installed.  This font contains a large range of Chinese/Japanese/Korean characters. | `0` |
+|`STARTUP_LOGFILE`| The location of where to write logs of the startup script responsible for installing & starting the Backblaze app. The default location goes to C:\ and so is also backed up to Backblaze. | `/config/wine/dosdevices/c:/backblaze-wine-startapp.log` |
```
Comment (traktuner) [inline README.md:121]: Please change the wording to:

The location for writing logs of the startup script, responsible for installing and starting the Backblaze app.
The default path is also backed up to Backblaze.

**Review by traktuner — CHANGES_REQUESTED** [review]
That is a good solution, thank you!
Please change the readme wording as in the comment.

File: README.md (hunk 1 above)
Comment (brokeh) [reply README.md]: Updated

**Review by traktuner — APPROVED** [review]

**Comment by Andrew-Brock** [comment]
Not sure why the CI is filing on this. Doesn't seem like anything related to the change

**Comment by traktuner** [comment]
> Not sure why the CI is filing on this. Doesn't seem like anything related to the change

yes, that's ecxpected.

**Pull request merged**